### DISCUSSION TITLE
Standalone statistics

### DISCUSSION THREAD

**Discussion by adk7712** [lowlighter/metrics#1467] [Q&A] [answered]
How can I remove this from the desired statistics which I require.

![image](https://github.com/lowlighter/metrics/assets/73314216/b96605ae-2540-4e47-b811-8ad8037a0b38)

**Comment by Nixinova** [comment, marked as answer]
Try option `base: []` (empty)

docs: https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#%EF%B8%8F-available-options
